src/Hosting/TestHost/src/Microsoft.AspNetCore.TestHost.csproj

```diff
@@ -12,8 +12,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Hosting" />
     <Reference Include="System.IO.Pipelines" />
-
-    <Compile Include="$(SharedSourceRoot)HostFactoryResolver\*.cs" />
+    <Reference Include="Microsoft.Extensions.HostFactoryResolver.Sources" />
```

Comment:
Why does TestHost reference this? It doesn't do Startup discovery.

Reply:
https://github.com/dotnet/aspnetcore/blob/master/src/Hosting/TestHost/src/WebHostBuilderFactory.cs#L14

Reply:
Hmm, that literally isn't used anywhere.